orchagent/portsorch.cpp

```diff
@@ -5381,6 +5404,58 @@ void PortsOrch::generateQueueMap()
     m_isQueueMapGenerated = true;
 }
 
+void PortsOrch::removeQueueMapPerPort(const Port& port)
+{
+    /* Remove the Queue map in the Counter DB */
+    vector<string> queueVector;
+    vector<string> queuePortVector;
+    vector<string> queueIndexVector;
+    vector<string> queueTypeVector;
+
+    for (size_t queueIndex = 0; queueIndex < port.m_queue_ids.size(); ++queueIndex)
+    {
+        std::ostringstream name;
+        name << port.m_alias << ":" << queueIndex;
+
+        const auto id = sai_serialize_object_id(port.m_queue_ids[queueIndex]);
+
+        queueVector.emplace_back(name.str());
+        queuePortVector.emplace_back(id);
+
+        string queueType;
+        uint8_t queueRealIndex = 0;
+        if (getQueueTypeAndIndex(port.m_queue_ids[queueIndex], queueType, queueRealIndex))
+        {
+            queueTypeVector.emplace_back(id);
+            queueIndexVector.emplace_back(id);
+        }
+    }
+
+    m_counter_db->hdel(COUNTERS_QUEUE_NAME_MAP, queueVector);
```

Comment:
can we use m_queueTable->hdel(),  m_queuePortTable->hdel(), m_queueIndexTable()->hdel(), m_queueTypeTable->hdel()?

Reply:
it's the same operation

on the deInitPort we also using removing in the same way:
m_counter_db->hdel(COUNTERS_PORT_NAME_MAP, alias);

Reply:
for consistency of ->set and ->hdel operations, we should use "m_queueTable", "m_queuePortTable" and "m_queueIndexTable".  
`m_counter_db->hdel(COUNTERS_PORT_NAME_MAP, alias);` can be changed too if you want to address in this PR.

Reply:
@zhenggen-xu  - Code was fixed following your remark!

Reply:
@zhenggen-xu - Your comment was addressed. Code was changed accordingly. Can you please review and approve ?